File: src/main/java/com/example/meditime/model/ClientMedsDescriptions.java
package com.example.meditime.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ClientMedsDescriptions {

    private List<String> medications;

    private String recommendations;

    private String interactions;

}

File: Mobile/app/src/main/java/com/example/medicationapp/model/ClientMedsDescriptions.kt


package com.example.medicationapp.model

/** A simple model class to hold descriptive details about a client's medications.
Contains a list of medication names, recommendations, and interaction information.
Provides getter methods for recommendations and interactions but does not expose setters or the medication list.
 **/



class ClientMedsDescriptions {
    private var medication: List<String> = emptyList()

    private var recommendations: String =""

    private var interactions: String = ""


    fun getRecommendations(): String{
        return recommendations
    }

    fun getInteractions():String {
        return interactions
    }
}